packages/cli/src/package.ts

```diff
@@ -102,11 +103,23 @@ function createEntrypoints(
   );
 }
 
+export type ExportsConditions = {
+  module: string | { worker?: string; browser?: string; default: string };
+  default: string;
```

Comment:
> Note that Preconstruct's support for the `exports` field does not currently include generating ESM compatible with Node.js. While ESM builds are generated, they are targeting bundlers, not Node.js or browsers directly so they use the `module` condition, not the `import` condition.

Are the ESM bundles themselves not compatible with Node.js? And that's why you're not including the `import` condition? Or is this only saying that the `import` condition is out of scope for this PR?
 What happens to the `import` and `require` conditions that I'm manually adding? Will they be clobbered? So many questions!

Reply:
> Are the ESM bundles themselves not compatible with Node.js?

Essentially yes because Node.js ESM has different semantics to the semantics commonly adopted by bundlers